examples/nlp/question_answering/conf/question_answering_squad_config.yaml

```diff
@@ -1,6 +1,7 @@
 # Question Answering with SQUAD
 name: &name QA
 
+pretrained_model: null # pretrained QAModel model from list_available_models()
```

Comment:
How is it different from pretrained_model_name?

Reply:
the pretrained_model_name is only for BERT, like 'bert-base-uncased', this is the name for the QAModel that you can instantiate from cloud

Reply:
I know this will require a lot of changes, but maybe using `pretrained_encoder` would make more sense and will reduce confusion?

Reply:
in the same line - in CV people are using "backbone" to denote the underlying encoder/model, has something similar started to emerge in NLP?

Reply:
@tkornuta-nvidia  i agree with you on naming the backbone "pretrained_encoder". But it will create alot of changes :(. I will add to my dotos